src/_guides/libraries/create-library-packages.md

```diff
@@ -77,12 +79,14 @@ package provides an easy way to create web servers using Dart,
 and is laid out in a structure that is commonly used for Dart
 library packages:
 
-{% asset libraries/shelf.png alt="shelf root directory contains example, lib, test, and tool subdirectories" %}
+<img 
+  src="/assets/img/libraries/shelf.png"
+  alt="shelf root directory contains example, lib, test, and tool subdirectories">
 
 Directly under lib, the main library file,
 `shelf.dart`, exports API from several files in lib/src.
-To avoid exposing more API than intended —
-and to give developers an overview of the entire public API of the package —
+To avoid exposing more API than intended -
+and to give developers an overview of the entire public API of the package -
```

Comment:
I figured out what happened. You converted em dash with surrounding white space into hyphens. Was there a particular reason for doing so with the kramdown update?

\cc @kwalrath

Reply:
@parlough That's interesting, completely missed that. I didn't do this deliberately, so now it's a question of what did and why. Unless this is something I did accidentally in this one file while we were debugging it yesterday. Are you able to find other occurrences?

Reply:
I couldn't seem to link to individual line differences, so I tagged the other ones in a review.

If they cause problems, feel free to switch them to --- which Kramdown seems to convert to em dash from looking at their documentation.